src/pages/api/modules/media-requests/index.ts

```diff
@@ -0,0 +1,165 @@
+import { getCookie } from 'cookies-next';
+import Consola from 'consola';
+import { NextApiRequest, NextApiResponse } from 'next';
+import { getConfig } from '../../../../tools/config/getConfig';
+
+import { MediaRequest } from '../../../../widgets/media-requests/media-request-types';
+import { ConfigAppType } from '../../../../types/app';
+
+const Get = async (request: NextApiRequest, response: NextApiResponse) => {
+  const configName = getCookie('config-name', { req: request });
```

Comment:
I don't think you need to use getCookie in an api route with the types in NextJS aha but it doesn't matter